### ISSUE TITLE
Terminal tab should surface split information in aria label

### ISSUE THREAD

**Issue by isidorn** [microsoft/vscode#122332]
Testing #122242

Have multiple terminal tabs open. In one tab have a couple of split terminals. Visually they look like they belong to a tree, however this information is not conveyed to the screen reader. Here are some ideas on how to improve this:
* Put `aria-level = 2` for elements that are inside the tree. Currently all elements have an aria-level = 1. If you prefer to leave it like that then the role of the whole thing should be a `list` and not a `tree`
* Modify aria label of an element in case he is part of a split

**Comment by Tyriar** [comment]
We're converting this into a list next iteration https://github.com/microsoft/vscode/issues/121745, I think we need to indicate to the screen reader that it's a split though. Not sure how the special characters are read out now.

**Comment by Tyriar** [comment]
@isidorn hmm, should we be surfacing the fact that it's a split terminal to the screen reader at all? I think it's purely a UI feature so it shouldn't make a difference if you can't see the UI?

**Comment by isidorn** [comment]
@Tyriar oh great that you moving to the list. You should have a correct role `list` then.
I would still convey the group number as part of the aria label, since we do the same for editors in editor groups. It is useful for users to know how things are grouped on the screen in case there are navigation actions like switch tab, or move focus inside a terminal tab. Just make sure that the group label is at the end of the list element aria label so it is easy to skip.

**Comment by Tyriar** [comment]
Good point 👍

**Comment by Tyriar** [comment]
Scoped this issue down, conversion to list will be handled in #121745

**Comment by isidorn** [comment]
I see the code changes which look good, however when I try this out the group is not applied to the aria label, it is the same as before.

I have a ZSH, one tab with one terminal, and another tabs with two terminals. And all their labels are just `zsh`

**Comment by meganrogge** [comment]
![inspect](https://user-images.githubusercontent.com/29464607/116594528-29f09600-a8d7-11eb-8995-3d9266453637.PNG)

When I inspect a split terminal tab label, it has the correct aria label.

**Comment by isidorn** [comment]
Oh but you are setting the `aria-label` on the wrong HTML element `label-name`
You should use the `List AccessibilityProvider` so the `aria-label` is properly set on the `monaco-list-row` element. It has to be on that element, since that element has focus and that is what the Screen Reader will read.
So I think it should be fixed here
https://github.com/microsoft/vscode/blob/main/src/vs/workbench/contrib/terminal/browser/terminalTabsWidget.ts#L67
